libraries/SRV_Channel/SRV_Channel.h

```diff
@@ -50,6 +50,7 @@ class SRV_Channel {
 
     typedef enum
     {
+        k_GPIO                  = -1,           ///< used as GPIO pin (input or output)
```

Comment:
maybe we should consider calling this k_disabled.

Reply:
i think k_GPIO is pretty clear